### PR TITLE
Post migration cleanup

### PR DESCRIPTION
**Pull request by supreeth7**
### What type of PR is this?
cleanup

### What this PR does / Why we need it?
Removes redundant and incorrect data from the repository

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -30,20 +30,22 @@ git remote add upstream https://github.com/openshift/backplane-cli.git
...
 
 ```bash
 git fetch upstream
-git checkout upstream/master
+git checkout upstream/main
 git tag -a ${VERSION} -m "release ${VERSION}"
 git push upstream $VERSION
 ```
 
+**Note:** We follow [semver](https://semver.org/) for versioning.
```
Comment (NautiluX) [inline docs/release.md:42]: probably worth mentioning that releases should start with a `v` for consistency. I think that's not mandatory in semver per se.

File: docs/release.md [hunk 2]
```diff
@@ -30,20 +30,22 @@ git remote add upstream https://github.com/openshift/backplane-cli.git
...
 
-Create a tag on the latest master.
+Create a tag on the latest main.
 
 ```bash
 git fetch upstream
-git checkout upstream/master
+git checkout upstream/main
 git tag -a ${VERSION} -m "release ${VERSION}"
 git push upstream $VERSION
```
Comment (NautiluX) [inline docs/release.md:39]: I had to use ssh to push to upstream.

**Review by NautiluX — CHANGES_REQUESTED** [review]

File: docs/release.md (hunk 2 above)
Comment (supreeth7) [reply docs/release.md:39]: It worked for me without having to use SSH.
Perhaps something was off with the `GITHUB_TOKEN`?

**Pull request merged**